src/openms/include/OpenMS/CHEMISTRY/EmpiricalFormula.h

```diff
@@ -282,6 +282,8 @@ namespace OpenMS
 
     /// remove elements with count 0
     void removeZeroedElements_();
+    
+    void sanitizeIfNotValidFormula(String& input_formula) const;
```

Comment:
needs underscore _ as protected member
add one line docu
maybe just use https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1String.html#a86864c88ee12eb322f1645f238c1dbe5

Reply:
changed it to trim() instead